app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
```

Comment:
it is a good practice to separate different types of imports with blank lines + arranging them alphabetically

Reply:
Hasn't been done yet